lcls-twincat-motion/Library/Library.tmc

```diff
@@ -1,7 +1,7 @@
(63)</Type><BitSize>512</BitSize><BitOffs>1024</BitOffs></SubItem><Hides><Hide GUID="{D91E046A-A488-4D27-8D43-0F3C40ED5081}" /><Hide GUID="{5DCEB2BC-E196-43AD-80B7-EBACF31A430B}" /><Hide GUID="{1B9FDDE4-B3B7-4F0F-AB14-24EDC2F643E7}" /><Hide GUID="{C1C52E30-BC0B-44CA-BF39-E2FE7F2D145C}" /></Hides></DataType><DataType><Name GUID="{56294066-FFF7-46F3-8206-FA06A30B13BA}" Namespace="PLC" TcBaseType="true">PlcTaskSystemInfo</Name><BitSize>1024</BitSize><SubItem><Name>ObjId</Name><Type GUID="{18071995-0000-0000-0000-00000000000F}">OTCID</Type><BitSize>32</BitSize><BitOffs>0</BitOffs></SubItem><SubItem><Name>CycleTime</Name><Type GUID="{18071995-0000-0000-0000-000000000008}">UDINT</Type><BitSize>32</BitSize><BitOffs>32</BitOffs></SubItem><SubItem><Name>Priority</Name><Type GUID="{18071995-0000-0000-0000-000000000005}">UINT</Type><BitSize>16</BitSize><BitOffs>64</BitOffs></SubItem><SubItem><Name>AdsPort</Name><Type GUID="{18071995-0000-0000-0000-000000000005}">UINT</Type><BitSize>16</BitSize><BitOffs>80</BitOffs></SubItem><SubItem><Name>CycleCount</Name><Type GUID="{18071995-0000-0000-0000-000000000008}">UDINT</Type><BitSize>32</BitSize><BitOffs>96</BitOffs></SubItem><SubItem><Name>DcTaskTime</Name><Type GUID="{18071995-0000-0000-0000-00000000000C}">LINT</Type><BitSize>64</BitSize><BitOffs>128</BitOffs></SubItem><SubItem><Name>LastExecTime</Name><Type GUID="{18071995-0000-0000-0000-000000000008}">UDINT</Type><BitSize>32</BitSize><BitOffs>192</BitOffs></SubItem><SubItem><Name>FirstCycle</Name><Type GUID="{18071995-0000-0000-0000-000000000030}">BOOL</Type><BitSize>8</BitSize><BitOffs>224</BitOffs></SubItem><SubItem><Name>CycleTimeExceeded</Name><Type GUID="{18071995-0000-0000-0000-000000000030}">BOOL</Type><BitSize>8</BitSize><BitOffs>232</BitOffs></SubItem><SubItem><Name>InCallAfterOutputUpdate</Name><Type GUID="{18071995-0000-0000-0000-000000000030}">BOOL</Type><BitSize>8</BitSize><BitOffs>240</BitOffs></SubItem><SubItem><Name>RTViolation</Name><Type GUID="{18071995-0000-0000-0000-000000000030}">BOOL</Type><BitSize>8</BitSize><BitOffs>248</BitOffs></SubItem><SubItem><Name>TaskName</Name><Type GUID="{18071995-0000-0000-0000-00010000003F}">STRING(63)</Type><BitSize>512</BitSize><BitOffs>512</BitOffs></SubItem><Hides><Hide GUID="{6A76D020-03A2-465C-A678-C341951E9EF3}" /><Hide GUID="{6F7D679F-72A0-4831-A7F1-085F839743ED}" /></Hides></DataType></DataTypes><Modules><Module GUID="{DED1545C-5FCD-48E2-B686-7D2C250A3641}" TcSmClass="TComPlcObjDef"><Name>Library</Name><CLSID ClassFactory="TcPlc30">{08500001-0000-0000-F000-000000000064}</CLSID><Licenses /><Contexts /><Parameters /><DataAreas /><Deployment /><EventClasses /><Properties><Property><Name>ApplicationName</Name><Value>Port_851</Value></Property><Property><Name>ChangeDate</Name><Value>2019-07-16T17:41:15</Value></Property><Property><Name>GeneratedCodeSize</Name><Value>217088</Value></Property><Property><Name>GlobalDataSize</Name><Value>57344</Value></Property></Properties></Module></Modules></TcModuleClass>
```

Comment:
Can't see what changed here effectively. I wonder if we should have a commit hook that runs an xml formatter on these files?

Reply:
I'm not sure how to set such a thing up but it's a good idea.